jwst/tweakreg/tweakreg_catalog.py

```diff
@@ -4,7 +4,8 @@
 
 
 def make_tweakreg_catalog(model, kernel_fwhm, snr_threshold, sharplo=0.2,
-                          sharphi=1.0, roundlo=-1.0, roundhi=1.0):
+                          sharphi=1.0, roundlo=-1.0, roundhi=1.0,
+                          brightest=None):
```

Comment:
Need to add peakmax to the argument list here, so that it's defined for use in line 72.

Reply:
Thank you!